coordinator/src/node.rs

```diff
@@ -266,7 +267,16 @@ impl Node {
         let maturity_time = trade_params.filled_with.expiry_timestamp;
         let maturity_time = maturity_time.unix_timestamp();
 
-        let fee_rate = self.settings.read().await.contract_tx_fee_rate;
+        let sats_per_vbyte = self
+            .inner
+            .fee_rate_estimator
+            .get(ConfirmationTarget::Normal)
+            .as_sat_per_vb()
+            .floor();
```

Comment:
I guess it should be always a round number but maybe _rounding_ makes more sense here?